frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -165,10 +172,17 @@ const ProjectListView: React.FC<IProps> = ({ filterable, title, mode }) => {
     const ReviewWorkflowStatuses = Object.keys(ReviewWorkflowStatus).map(
       (k: string) => (ReviewWorkflowStatus as any)[k],
     );
-    if (row.statusCode === ReviewWorkflowStatus.PropertyReview) {
+    if (
+      row.statusCode === ReviewWorkflowStatus.PropertyReview &&
+      adminOrCanApprove(keycloak, row)
+    ) {
       history.push(`/dispose/projects/assess/properties?projectNumber=${row.projectNumber}`);
     } else if (ReviewWorkflowStatuses.includes(row.statusCode)) {
-      history.push(`/dispose/projects/approved?projectNumber=${row.projectNumber}`);
+      if (keycloak.hasClaim(Claims.ADMIN_PROJECTS)) {
+        history.push(`/dispose/projects/approved?projectNumber=${row.projectNumber}`);
```

Comment:
How are we handling when users manually return to a page?  Can they make edits when they shouldn't be allowed?

Reply:
Users:
1) Can't get to any of the draft pages at all after submission. Those pages throw errors when manually navigated to after submission
2) Manually navigating back to the project review page will load that page, but none of the buttons are clickable.